CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 
 ## Compatibility Notes
```

Comment:
I changed it a bit to make it even more explicit. Could you also add an entry to fixed issues?